sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -846,7 +846,7 @@ org.apache.spark.SparkIllegalArgumentException
   "sqlState" : "22023",
   "messageParameters" : {
     "charset" : "WINDOWS-1252",
-    "charsets" : "UTF-16LE, UTF-8, UTF-32, UTF-16BE, UTF-16, US-ASCII, ISO-8859-1",
+    "charsets" : "utf-8, utf-16be, iso-8859-1, utf-16le, utf-16, utf-32, us-ascii",
```

Comment:
May I ask why the order is different?

Reply:
this is a Set, maybe we shall sort it before throwing